src/hexdeck.gleam

```diff
@@ -214,6 +216,7 @@ fn view_embed_wrapper(
               ]),
               html.input([
                 attribute.class("bg-transparent flex-1"),
+                attribute.attribute("autocapitalize", "off"),
```

Comment:
Don't feel like you need to change this, but I have a personal habit of importing the `attribute` unqualified so it'd be `attribute("autocapitalize", "off")`. Not a blocker though!

Reply:
i don't mind changing it, when it adapts more to your style 😉